### ISSUE TITLE
Remove unused Root CAs

### ISSUE THREAD

**Issue by emanuelb** [phpmyadmin/docker#129]
phpmyadmin use outgoing connections only to *.phpmyadmin.net websites (See httpRequest function) which are signed by LetsEncrypt, thus it's possible to remove unused Root CAs from the container.

**Comment by nijel** [comment]
Captcha can be also enabled, so we need the ones for Google as well.

Maybe I'm missing some other remote service which could be used, it should be deeper checked prior to removal...

**Comment by emanuelb** [comment]
Right, it's called from 3rd-party code (recaptcha library) thus need to have look at other libs network usage also.
Regarding Google a pinning probably can be accomplished as they use only small subset of Root CAs (4 according to https://www.imperialviolet.org/2011/05/04/pinning.html which point that it's located in chromium source-code as well, to check in case they changed the list)

**Comment by emanuelb** [comment]
I Reported the issue to Recaptcha Library:
https://github.com/google/recaptcha/issues/165

**Comment by emanuelb** [comment]
The mechanism need to rely on upstream pinning (thus breakage will be likely upstream fault,not need to maintain the white-list here), for phpmyadmin that mean parse the certificates in /libraries/certs/ directory.
